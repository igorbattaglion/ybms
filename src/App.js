import React from 'react';
import { Provider } from 'react-redux';
import store from './store'
import { BrowserRouter } from 'react-router-dom'
import Routes from './routes'
import './styles/style.css'

function App() {
  return (
    <Provider store={store}>
      <BrowserRouter>
        <Routes />
      </BrowserRouter>
    </Provider>
  );
}

export default App;
