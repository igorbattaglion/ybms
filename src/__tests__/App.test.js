import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('Testing App Component', () =>{
  it('should render correctly', () =>{
    const wrapper = shallow(
      <App/>
    );

    expect(wrapper).toMatchSnapshot()
  })
})