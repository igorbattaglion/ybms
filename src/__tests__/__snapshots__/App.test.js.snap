// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing App Component should render correctly 1`] = `
<div
  className="App"
>
  <h1>
    Hello World
  </h1>
  <h2>
    Hello World!
  </h2>
</div>
`;
